demo/web/src/main/notebooks/02 Stream and Batch Together.md

```diff
@@ -142,11 +142,11 @@ You can use Deephaven for **time-series and relational** joins on static tables,
 Please note that you do not have to think about whether a named_table happens to be updating (stream-like) or static (batch-like). The methods are common between the two.
 
 ```python
-join_stream_batch = agg_streaming.renameColumns("Price_Streaming = Price", "Avg_Price_Streaming = Avg_Price")\
-  .naturalJoin(agg_batch, "Exchange, Instrument", "Date_Batch = Date, Price_Batch = Price, Avg_Price_Batch = Avg_Price")\
+join_stream_batch = agg_streaming.rename_columns(["Price_Streaming = Price", "Avg_Price_Streaming = Avg_Price"])\
+  .natural_join(right=agg_batch, on=["Exchange, Instrument", "Date_Batch = Date, Price_Batch = Price, Avg_Price_Batch = Avg_Price"])\
   .formatColumns("Date = `IVORY`")\
```

Comment:
Check that formatColumns works.  Last I checked it was not wrapped or I missed it

Reply:
It is not currently wrapped.  Jianfeng is working on a V2 wrapper now.  When that's merged into main, I'll update these lines and this notebook will be ready for a merge.